maintainer/conda/MDAnalysis/meta.yaml

```diff
@@ -1,79 +1,56 @@
 package:
   name: mdanalysis
   # This has to be changed after a release
-  version: "0.15.0"
+  version: "0.16.0dev"
 
 source:
-   # to build from source you can speficy the path to the code directly.
-   # path: ../../../
-   # This ensures that you will build from a clean checkout
    git_url: https://github.com/MDAnalysis/mdanalysis
-   # git_branch: master
-   git_tag: release-0.15.0
+   git_branch: develop
+   # git_tag: release-0.15.0
 
 # specify the subversion of the conda package. Defaults to 0. If you make
 # changes to the conda package alone with out an increase in the source package
 # version increase this number. Should be reset to 0 after a new release
 build:
-  number: 1
+  number: 0
```

Comment:
This is thought that people can make their own builds and then they should start with a 0. If we do our own builds we can change this scripted. We don't need to commit these changes everytime we make our own nightly build.